packages/migrate/src/__tests__/fixtures/schema-only-sqlserver/prisma/invalid-url.prisma

```diff
@@ -1,4 +1,4 @@
 datasource my_db {
   provider = "sqlserver"
-  url      = "sqlserver://doesnotexist:1433;database=tests-migrate;user=SA;password=Pr1sm4_Pr1sm4;trustServerCertificate=true;encrypt=DANGER_PLAINTEXT"
+  url      = "sqlserver://doesnotexist:1433;database=tests-migrate;user=SA;password=Pr1sm4_Pr1sm4;trustServerCertificate=true;
```

Comment:
```suggestion
  url      = "sqlserver://doesnotexist:1433;database=tests-migrate;user=SA;password=Pr1sm4_Pr1sm4;trustServerCertificate=true;"
```

Reply:
Should this have failed in the tests?

Reply:
Looks like this one is not tested on SQL Server indeed.